api/lib/schema.js

```diff
@@ -316,6 +323,21 @@ const typeDefs = gql`
     depositThreshold: String # BigNumber
   }
 
+  type GovernanceLink {
+    title: String
+    link: String
+    type: String
+  }
+
+  type GovernanceOverview {
+    totalStakedAssets: Float
+    totalVoters: Int
+    treasurySize: Float
+    recentProposals: [Proposal]
+    topVoters: [String] @cacheControl(maxAge: 21600)
```

Comment:
this would only cache this property, but to answer the request, u still need to do all the calls, so topVoters would still be calculated every time. we should cache governance overview completely and remove "recentProposals". "recentProposals" should be a non cached request.

Reply:
Instead of using "recentProposals" I suggest we just use the already existing "proposals" query.